openmc/mesh.py

```diff
@@ -181,6 +183,21 @@ def __repr__(self):
         string += '{0: <16}{1}{2}\n'.format('\tPixels', '=\t', self._width)
         return string
 
+    def cell_generator(self):
+        """Generator function to traverse through every [i,j,k] index
+        of the mesh.
+
+        """
+        if len(self.dimension) == 2:
+            for x in range(self.dimension[0]):
+                for y in range(self.dimension[1]):
+                    yield [x + 1, y + 1, 1]
+        else:
+            for x in range(self.dimension[0]):
+                for y in range(self.dimension[1]):
+                    for z in range(self.dimension[2]):
+                        yield [x + 1, y + 1, z + 1]
```

Comment:
Yeah, so the mesh indices technically begin at 1.  So, this is true to that.  The instance where you see me take off the 1 is because I am using the produced [i,j,k] to index a numpy array which is 0-based.